### PR TITLE
Feature/country state dropdown

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the code changes for Country and state dropdown list in checkout form.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/checkoutform.html [hunk 1]
```diff
@@ -121,16 +121,31 @@
 					<p class ='error-message'></p>
 					</div>
 
-                    <div>
-					<!-- <input type="text" 
+                    <!--<div>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/checkoutform.html]: @nitheshkumar-hcl : can we remove unwanted code from here

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js [hunk 2]
```diff
@@ -272,17 +322,22 @@ function onSaveNDeliver() {
 	if (!doValidation && checkoutcartId) {
 		$('.empty-cartid').text("");
 		getUserDeatils['cartId'] = checkoutcartId;
-		getUserDeatils['region'] = "MH";
-		getUserDeatils['region_id'] = 0;
-		getUserDeatils['country_id'] = 'IN';
-		getUserDeatils['region_code'] = 'MH';
+
+        console.log("getUserDeatils", getUserDeatils);
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js]: @nitheshkumar-hcl : pl remove console logs

File: core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	private static final long serialVersionUID = 4016057296495129474L;
+	private static final Logger LOG = LoggerFactory.getLogger(CountryStateServlet.class);
+
+	@Reference
+	private ShippingInfoService shippingInfoService;
+
+	protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
+			throws ServletException, IOException {
+
+		LOG.info("inside CountryState Servlet  doGET method");
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java]: @nitheshkumar-hcl : remove log.info or change to log.debug

**Review by rishabh-behl — COMMENTED** [review]
@nitheshkumar-hcl : pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/checkoutform.html (hunk 1 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/checkoutform.html]: Committed the changes as per the review comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js (hunk 2 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js]: Committed the changes as per the review comments

File: core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java (hunk 3 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java]: Committed the changes as per the review comments

Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/CountryStateServlet.java]: Committed the changes as per the review comments

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**